### PR TITLE
GUACAMOLE-926: Document connection batch import.

### PR DESCRIPTION
**Pull request by jmuehlner**
I just copied the file format documentation from the webapp.

### CODE REVIEW

File: src/images/batch-import-start.png
Comment (mike-jumper) [inline src/images/batch-import-start.png:1]: Something happened to this screenshot that added a single red dot in a region of transparency along the left border.

Comment (jmuehlner) [reply src/images/batch-import-start.png:1]: Oops - fixed!

File: src/images/batch-import-admin-link.png
Comment (mike-jumper) [inline src/images/batch-import-admin-link.png:1]: Sorry - made some more work for you with the merge of apache/guacamole-client#1012 (moved the "Import" button such that "New Connection" is back where it used to be). Can you update this screenshot to match?

Comment (jmuehlner) [reply src/images/batch-import-admin-link.png:1]: Sure

**Review by mike-jumper — APPROVED** [review]

**Pull request merged**